_resources/port1.0/group/java-1.0.tcl

```diff
@@ -59,7 +59,9 @@ namespace eval java {
             # as required, as currently these are the only ones supporting arm.
             # To be reviewed as support for arm comes for the other versions.
             # Following regex matches openjdk<version> only.
-            if { [option configure.build_arch] eq "arm64" &&
+            # Keep in mind that configure.build_arch isn't available here
+            global os.arch
+            if { ${os.arch} eq "arm" &&
```

Comment:
While `[option configure.build_arch]` looks incorrect, and needs to be changed... `configure.build_arch` isn't accessible, after declaring as a global...?

Reply:
@mascguy yes, it doesn't. It was a while until I've found `os.arch` :)